recipes/actions-runner/deactivate.d/actions-runner-force-node16.sh

```diff
@@ -0,0 +1,3 @@
+unset ACTIONS_RUNNER_FORCE_ACTIONS_NODE_VERSION
+
+export ACTIONS_RUNNER_FORCE_ACTIONS_NODE_VERSION=$_ACTIONS_RUNNER_FORCE_ACTIONS_NODE_VERSION
```

Comment:
I thought this should be sufficient to cleanup the env vars? Will have to do some testing 🤔

Reply:
What if I don't have this env set initially? Then the env ends up with an empty variable it didn't have iiuic.